src/common/types/store-data/unified-data-interface.ts

```diff
@@ -48,11 +48,15 @@ export interface InstancePropertyBag {
 
 export type StoredInstancePropertyBag = InstancePropertyBag;
 
+export type IdentifiersStoredInstancePropertyBag = {
+    shortName: string;
```

Comment:
For some properties that are agnostic of platforms and we are sure that they would exist in all platforms, we feel that we should have more strong typing on the property. For example, shortName is one of them.